packages/core/src/components/footer/footer.tsx

```diff
@@ -18,6 +18,9 @@ export class TdsFooter {
   /** Mode variant of the component, based on current mode. */
   @Prop() modeVariant: 'primary' | 'secondary' = null;
 
+  /** Compact design of the component. */
+  @Prop() compactDesign: boolean = false;
+
```

Comment:
This prop is not used anywhere, right?

Reply:
yeah, after commiting suggestion it was left there,, now commited new one, it should be fine